services/object/schemas/v1/download-object-request.yml

```diff
@@ -0,0 +1,17 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Download Object Request"
+description: |
+  Download object request. See [Transport Protocols](https://docs.taskcluster.net/docs/reference/platform/object/transport-protocols) for more detail.
+type:               object
+properties:
+  acceptProtocols:
+    type: array
```

Comment:
It is a good question whether the download method should also carry parameters. Currently indeed only a string is assumed, with the name of the download method to use, rather than an object with parameters.

So long as `acceptDownloadMethods` is an `enum`, in future json objects could be added, so that parameterised download methods could be supported, like in your example, but for the first revision, this probably isn't needed, and I think since this is controlled at a schema level, it would be possible to enhance this in future if a new download method takes parameters.